modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -22,6 +22,7 @@ import {
   ClientCore,
   computeProposalStatusFilter,
   ContextPlugin,
+  etherToUnitInterval,
```

Comment:
interval? Something like `ratio` feels more meaningful. Maybe `parseEtherRatio()` or related?

Reply:
Yes i used unit interval bcause is the set of numbres between 0 and 1, but I wasnt super convinced about the name